src/components/Banner/Banner.scss

```diff
@@ -13,3 +13,13 @@
     padding-bottom: 4.444rem;
   }
 }
+
+.content-banner {
+  margin-left: calc(1 / 0.9 / 2 * -10%);
+  width: calc(1 / 0.9 * 100%);
+
+  @include from($desktop) {
+    margin-left: calc((100vw - 100%) / -2 - 8px);
```

Comment:
What's this `- 8px` for? At the moment there seems to be a narrow gap in the right side of the exercise container and it seems to be because of this.

Reply:
It's accounting for the sidebar, but the width is platform dependent. Let me try to get rid of that altogether by not trying to escape containing elements with negative margins.

Reply:
Tried to go without negative margins [here](https://github.com/vvilhonen/fullstack-hy2020.github.io/commit/6754e12f4c4e03d05bc81ebdb300b4b1d0349dcc), but it seems hopeless. 

Scrollbar width is now calculated with JS on mount and `resize` events and stored to a CSS variable that's being used in the calculation of the negative margin. I get no gaps or unnecessary horizontal scrollbars on windows 10 (chrome, FF) at least.

[Live demo](http://improvements-demo.s3-website.eu-north-1.amazonaws.com/) is also updated to reflect the change.

Reply:
Great, that takes care of the scrollbar! There still seems to be some buggy stuff going on with a bit narrower screens. Somehow all this feels overly complex. 🤔

<img width="993" alt="Screenshot 2020-07-14 at 11 03 06" src="https://user-images.githubusercontent.com/3078411/87403214-3fedce00-c5c5-11ea-9e28-4099b9a74b10.png">

Reply:
This sure is tricky to get right with 100% CSS, but I think I finally got it now. There's no magic constants anymore or any JS in this redesign and the structure looks way more intuitive in inspector. 

Tested on 200-2000px width viewports on Edge, FF, Chrome on windows 10 and safari on iOS.

The same old [demo env](http://improvements-demo.s3-website.eu-north-1.amazonaws.com) is also updated.

If there are no new issues and this is desired to be merged, let me know and I'll clean up the intermediate versions from the branch history into more logical ones before that.

Reply:
In my opinion, everything looks good and this ready to be merged. Let me know when you're done with the cleanup and I'll press the button.

Thanks a lot for your effort, this is a huge improvement! 🙏